environment.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
   - aiohttp
   - aiomysql
   - aiosqlite
+  - asyncache
```

Comment:
I'm not sure there is enough code in this package to justify it as a dependency. It also doesn't seem to be that widely used or actively maintained.